src/main/java/dk/cngroup/lentils/controller/TeamController.java

```diff
@@ -0,0 +1,79 @@
+package dk.cngroup.lentils.controller;
+
+import dk.cngroup.lentils.entity.Team;
+import dk.cngroup.lentils.service.TeamService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.validation.Valid;
+import java.util.Optional;
+
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+
+@Controller
+@RequestMapping("/team")//http://localhost:8080/team/add
+public class TeamController {
+    private final TeamService teamService;
+
+    private final String VIEW_PATH = "team/";
+
+    public TeamController(TeamService teamService) {
+        this.teamService = teamService;
+    }
+
+    @RequestMapping(value = "/add", method = GET)
```

Comment:
Jen koment: Ještě je tam anotace @GetMapping, @PostMapping, tim se to da jeste krapet zkratit.